packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -154,60 +154,50 @@ export const Webchat = forwardRef((props, ref) => {
     menuIsOpened ? setMenuIsOpened(false) : setMenuIsOpened(true)
   }
 
+  const checkBlockInput = input => {
+    if (!Array.isArray(props.blockInputs)) return
+    for (let rule of props.blockInputs) {
+      if (rule.match.some((l, m) => l.test(input.data))) {
```

Comment:
what is `l` in this context? Maybe would be interesting to rename it as `regex`.
`if (rule.match.some((regex, m) => regex.test(input.data)))`. In this way I think that would be more clear.
Also, the second parameter of the function is not used at all. To explicitly say that this param is not used (with an underscore), you can pass the parameters as (regex, _), or even better, just use `regex => regex.test`.